src/button.tsx

```diff
@@ -20,3 +22,85 @@ export const ProtoButton = React.forwardRef<HTMLButtonElement, JSX.IntrinsicElem
     {...rest}
   >{children}</button>,
 );
+
+/** The kind of buttons a "Button" can be. Used for styling */
+export type Kind = "normal" | "danger" | "call-to-action";
+
+type ButtonProps = JSX.IntrinsicElements["button"] & {
+    kind?: Kind;
+    extraCss?: Interpolation<Theme>;
+};
+
+/** A styled button */
+export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ kind = "normal", extraCss, children, ...rest }, ref) => (
+    <button ref={ref} type="button" css={css(kind, extraCss)} {...rest}>{children}</button>
+  ),
+);
+
+/**
+ * Returns css for different types of buttons.
+ * Comes in the kinds "normal", "danger" and "happy".
+ */
+const css = (kind: Kind, extraCss: Interpolation<Theme> = {}): Interpolation<Theme> => {
+  const config = useAppkitConfig();
+
+  const notDisabledStyle = match(kind, {
+    "normal": () => ({
+      border: `1px solid ${config.colors.neutral40}`,
+      color: config.colors.neutral90,
+      "&:hover, &:focus-visible": {
+        border: `1px solid ${config.colors.neutral60}`,
+        backgroundColor: config.colors.neutral15,
+      },
+    }),
+
+    "danger": () => ({
+      border: `1px solid ${config.colors.danger4}`,
+      color: config.colors.danger4,
+      "&:hover, &:focus-visible": {
+        border: `1px solid ${config.colors.danger5}`,
+        backgroundColor: config.colors.danger4,
+        color: config.colors.danger0BwInverted,  // danger0BwInverted
+      },
+    }),
+
+    "call-to-action": () => ({
+      border: `1px solid ${config.colors.happy5}`,
+      color: config.colors.happy4BwInverted, // happy0BwInverted
+      backgroundColor: config.colors.happy4,
+      "&:hover, &:focus-visible": {
+        border: `1px solid ${config.colors.happy6}`,
+        backgroundColor: config.colors.happy5,
+        color: config.colors.happy5BwInverted,  // happy1BwInverted
```

Comment:
I haven't tested these colors but if the goal is to get close to Tobiras buttons, I think other colors should be used?

```suggestion
      border: `1px solid ${config.colors.happy8}`,
      color: config.colors.happy7BwInverted,
      backgroundColor: config.colors.happy7,
      "&:hover, &:focus-visible": {
        border: `1px solid ${config.colors.happy9}`,
        backgroundColor: config.colors.happy8,
        color: config.colors.happy8BwInverted,
```